### DISCUSSION TITLE
Проблема с импортом level

### DISCUSSION THREAD

**Discussion by SquareSN** [PavelBlend/blender-xray#637] [Q&A] [answered]
Выбираю в импорте "level", в открывшемся окне тоже выбираю нужный файл. 
![level](https://github.com/PavelBlend/blender-xray/assets/133961102/31fe38f3-11d3-4c9c-a13c-6ea20b8f9f23)

По итогу блендер думает минуту, и просто вылетает без ошибки. Помогите пожалуйста, очень нужно импортировать локацию. (Blender последней версии)

**Comment by SamArtGit** [comment, marked as answer]
Обсуждалось здесь #622 и здесь #626.

**Reply by SquareSN** [reply]
Спасибо! Поставил блендер 5.4.1. Проблема не решилась, оказалось, что у меня было пустое поле в параметрах аддона "objects folder" прописав к нему вручную путь проблема ушла на блендере 5.4.1. На самой новой версии не работает.
